### DISCUSSION TITLE
Spotify Player support arrives in v2.1.6

### DISCUSSION THREAD

**Discussion by maboa** [hyperaudio/hyperaudio-lite#179] [Announcements]
A beta version of the Spotify player is now available, unfortunately the [Spotify iFrame API ](https://developer.spotify.com/documentation/embeds/references/iframe-api) is somewhat limited and so we don't quite have the full functionality of other players.

The API limits us to:
1. A seek accuracy of nearest second
2. An update frequency of one second (although a workaround is provided)
3. Playing a file without previous interaction will always play from start
  ie – a shared selection will highlight but not start at the start of that selection. 

Here's an example of it in action https://hyperaudio.github.io/hyperaudio-lite/spotify.html Keen to hear what people think, especially @andreashorn who filed the issue requesting it. :)

**Comment by heyhil** [comment]
Hi maboa. curious is there are more specific instructions somewhere for this? would love to use it. thank you!
